### ISSUE TITLE
Reproducing the IC downstream task

### ISSUE THREAD

**Issue by mrezasoltani** [s3prl/s3prl#432]
Hi,

I am trying to reproduce the IC accuracy results you have reported in the SUPERB paper using Wave2Vec2, Hubert, etc. It is not clear for me what architecture/hyperparameters you have used for the IC downstream task. Can you please explain different hyperparameter setup? Specifically:

1 - Did you use only the hidden features from the last layer of the upstream task (e.g., Wave2Vec2)?
2 - Did you fine tune the upstream with IC model (downstream), or you just freeze the upstream model?
3 - What is the architecture you have used for IC task? layers, activations, etc.?
4 - The optimization algorithm and its hyperparameters (learning rate, schedulers, etc.)?

It seems the model.py in "downstream)/fluent commands/" folder is not matched with the IC experiment. I think you have used the code in "downstream)/model.py", instead?

Thanks

**Comment by leo19941227** [comment]
Hi,

1. no we use all layers and do the weighted-sum over them like ELMo
2. no we freeze the upstream model
3. a simple 2-layer linear model with mean pooling, which is the `UtteranceLevel` model in the `downstream/model.py`
4. http://140.112.21.28:8000/ic/

The `model.py` in `downstream/fluent_commands` is not used in SUPERB

**Comment by mrezasoltani** [comment]
> 

Thanks for your answer. Just one more question: the results you have reported in Table 2 of the SUPERB paper are the test accuracy, or the val accuracy?

**Comment by leo19941227** [comment]
Hi,

test accuracy

**Comment by mrezasoltani** [comment]
> Hi,
> 
> test accuracy

Thanks
